### DISCUSSION TITLE
essay/2024/05/08/developer-podcasts

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#210] [Comments]
# essay/2024/05/08/developer-podcasts

운전할 때 팟캐스트 듣는 걸 좋아합니다. 개발자 팟캐스트.

https://jeho.page/essay/2024/05/08/developer-podcasts.html

**Comment by developedbyDK** [comment]
좋은 말씀 잘 들었습니다. 1인 개발을 하고 있는 사람으로서 큰 동기부여가 되었습니다. 감사합니다.

**Reply by BenjaminKim** [reply]
이야기 들어주셔서 고맙습니다. 동기부여가 된다 말씀해주시니 힘이 납니다.

**Comment by yongbeom76** [comment]
영감도 얻고 격려 받고 갑니다. 경험을 나눠주시니 저도 새로운 목표 세우고 도전해보겠습니다. 아직은 도전할 상황은 아니지만 방향설정은 될거 같아요!

**Reply by BenjaminKim** [reply]
응원하겠습니다. 인사 남겨주셔서 고맙습니다.

**Comment by bookseal** [comment]
요즘 개발자 팟캐스트도 뭐가 있나 살펴보겠습니다.

**Reply by BenjaminKim** [reply]
요즘엔 영어 공부겸 영어 개발 팟캐스트도 들어보고 있는데... 안 들려서 죽겠네요. 후.

**Reply by yongbeom76** [reply]
영어공부 도와드릴수 있습니다. 성인영어강사입니다. 관심있으시면 연락 주세요. 전공은 컴공했습니다^^ kakao: yongbeom76
